annotationPipeline/src/test/resources/expected/output_contains_original_genomic_location.expected.txt

```diff
@@ -0,0 +1,6 @@
+#genome_nexus_version: null
+#isoform: mskcc
+Hugo_Symbol	Entrez_Gene_Id	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Consequence	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	Validation_Status	Mutation_Status	Sequencing_Phase	Sequence_Source	Validation_Method	Score	BAM_File	Sequencer	t_ref_count	t_alt_count	n_ref_count	n_alt_count	HGVSc	HGVSp	HGVSp_Short	Transcript_ID	RefSeq	Protein_position	Codons	Exon_Number	IGNORE_Genome_Nexus_Original_Chromosome	IGNORE_Genome_Nexus_Original_End_Position	IGNORE_Genome_Nexus_Original_Reference_Allele	IGNORE_Genome_Nexus_Original_Start_Position	IGNORE_Genome_Nexus_Original_Tumor_Seq_Allele1	Annotation_Status
+PIK3CA	5290		GRCh37	3	178916927	178916939	+	protein_altering_variant	In_Frame_Del	DEL	TAGGCAACCGTGA	G	G																								ENST00000263967.3:c.314_326delinsG	p.Val105_Glu109delinsGly	p.V105_E109delinsG	ENST00000263967	NM_006218.2	105	gTAGGCAACCGTGAa/gGa	2/21	3	178916939	TAGGCAACCGTGA	178916927	G	SUCCESS
+EGFR	1956		GRCh37	7	55220240	55220240	+	splice_region_variant,synonymous_variant	Splice_Region	SNP	G	T	T																								ENST00000275493.2:c.630G>T		p.X210_splice	ENST00000275493	NM_005228.3	210	ctG/ctT	6/28	7	55220240	G	55220240	T	SUCCESS
+EGFR	1956		GRCh37	7	55242468	55242486	+	protein_altering_variant	In_Frame_Del	DEL	ATTAAGAGAAGCAACATCT	AGCAA	GCAA																								ENST00000275493.2:c.2238_2256delinsGCAA	p.Leu747_Ser752delinsGln	p.L747_S752delinsQ	ENST00000275493	NM_005228.3	746	gaATTAAGAGAAGCAACATCT/gaGCAA	19/28	7	55242486	AATTAAGAGAAGCAACATCT	55242467	AGCAA	SUCCESS
```

Comment:
In this example the input file doesn't have `Tumor_Seq_Allele2` column, do we want to add `IGNORE_Genome_Nexus_Original_Tumor_Seq_Allele2` column with empty value? Or better to not adding it?